contrib/jenkins_tests/vg.sh

```diff
@@ -32,6 +32,7 @@ if [ ! -z "$(do_get_ip 'eth')" ]; then
 fi
 test_list="tcp:--tcp udp:"
 test_lib=${vg_dir}/install/lib/${prj_lib}
+test_lib_env="XLIO_MEM_ALLOC_TYPE=ANON XLIO_MEMORY_LIMIT=512MB XLIO_TX_BUFS=4000 XLIO_RX_BUFS=4000 XLIO_TX_WRE=2000 XLIO_RX_WRE=2000 XLIO_TSO=off XLIO_LRO=off XLIO_STRQ=off "
```

Comment:
1. Remove XLIO_TX_BUFS and XLIO_RX_BUFS since they're deprecated and will be removed in the future releases. These parameters trigger fallback mechanism which overwrites XLIO_MEMORY_LIMIT.
2. Is there any issue with strq configuration? Can we check XLIO_STRQ=on since this is default and more expected configuration? Or use several configurations and loop over them.

Reply:
Main issue with valgrind is huge memory used by process. So any techniques that reduce process memory usage during verification should help.
I set all suspicions parameters and probably some of them are redundant.

Reply:
I see for example:
```
 XLIO INFO   : Memory limit                   512 MB                     [XLIO_MEMORY_LIMIT]
 XLIO INFO   : ---------------------------------------------------------------------------
 XLIO WARNING: Requested XLIO_STRQ_NUM_STRIDES * XLIO_RX_WRE > Maximum CQE per CQ (4194304). Decreasing XLIO_RX_WRE to 256
```
in case ` XLIO_MEM_ALLOC_TYPE=ANON XLIO_MEMORY_LIMIT=512MB XLIO_TX_WRE=2000 XLIO_RX_WRE=2000`

Reply:
Striding RQ creates large buffers (e.g. 1MB) and `XLIO_RX_WRE` needs to be much smaller than for regular RQ. Anyway, this can be improved separately to cover different code paths.